### DISCUSSION TITLE
macOS ventura, Big Sur adapter app is damaged after enabling SIP solution

### DISCUSSION THREAD

**Discussion by HDV998** [chris1111/Wireless-USB-Big-Sur-Adapter#244] [General]
hi, i discovered this newly that if you want to use the adapter with SIP enabled in macOS Ventura, and get an error that the app is damaged and telling you to move it to the bin, just go to system settings, then security and privacy scroll down and allow to open the app anyway, enter the password and everything works just fine after that!

**Comment by chris1111** [comment]
Yea or simply disable gatekeeper

sudo spctl --master-disable
